include/Gaffer/Context.h

```diff
@@ -254,6 +254,15 @@ class GAFFER_API Context : public IECore::RefCounted
 
 				const Context *context() const { return m_context.get(); }
 
+				// Forward compatibility with Gaffer 0.60
```

Comment:
I liked the "this allows projects..." bit from the other comment - can we copy that here too?